src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const RENTAL_DAY_VALUE = 40;
+  const DISCAUNT_SHORT_TERM = 20;
```

Comment:
There is a spelling mistake in DISCAUNT_SHORT_TERM, it should be DISCOUNT_SHORT_TERM.